services/core/shared/core/compat/sdk_v2/accounts.js

```diff
@@ -73,5 +79,25 @@ const getAccounts = async params => {
 	return result;
 };
 
+const getMultisignatureGroups = async account => {
+	const result = await coreApi.getMultisignatureGroups(parseAddress(account.address));
+	return isProperObject(result) && Array.isArray(result.data)
+		? result.data[0]
+		: [];
```

Comment:
Recommend to make it a single liner, if the linter permits:

```suggestion
	return isProperObject(result) && Array.isArray(result.data) ? result.data[0] : [];
```

else try:

```suggestion
	return isProperObject(result) && Array.isArray(result.data)
		? result.data[0] : [];
```

Reply:
Done!